gcalcli/validators.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import absolute_import
+
+import re
+
+from builtins import input
+
+from gcalcli.utils import valid_override_colors, REMINDER_REGEX
+from gcalcli.exceptions import ValidationError
+
+from dateutil.parser import parse
+
+
+def get_input(printer, prompt,  validator_func):
```

Comment:
nit: two spaces in front of validator_func (but really, I'm surprised that flake8 didn't catch this!)